### PR TITLE
[pt] Add Explore You App in Portuguese

### PR DESCRIPTION
**Pull request by jailton**
Ref: #13939

Translate Explore Your App files in Portuguese:
-  content/pt/docs/tutorials/kubernetes-basics/explore/_index.md
-  content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html
-  content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html

Update link to Explore Your App Portuguese translation:
- content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

Preview: https://deploy-preview-26556--kubernetes-io-master-staging.netlify.app/pt/docs/tutorials/kubernetes-basics/explore/

/language pt

### CODE REVIEW

**Comment by jcjesus** [comment]
Great @jailton thanks for your contribution!
/approved

File: content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <div class="katacoda__alert">
+                Para interagir com o Terminal, por favor, use a versão para desktop ou table.
+            </div>
+
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/4" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/expose/expose-intro/" role="button">Continue para o Módulo 4<span class="btn__next">›</span></a>
```
Comment (jcjesus) [inline content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html:32]: Hi Jailton, here need include "/pt/" into the path to not break the URL.

File: content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                <div class="content__box content__box_fill">
+                    <p><i>Um nó é uma máquina operária do Kubernetes e pode ser uma VM ou máquina física, dependendo do cluster. Vários Pods podem ser executados em um nó.</i></p>
+                </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-interactive/" role="button">Inciar o Tutorial Interativo<span class="btn__next">›</span></a>
```
Comment (jcjesus) [inline content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html:134]: Hi, here again, need to write the prefix Portuguese folder like as "/pt" to fix the pah.
Thanks

**Review by jcjesus — COMMENTED** [review]
Hi Jailton, please, I think is need anothers fixes to approve this PR.
Thanks!

File: content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html (hunk 1 above)
Comment (jailton) [reply content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html:32]: Hi @jcjesus! There is no **Expose Your App Publicly** translation to Portuguese. I'll translate this section next week.

File: content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html (hunk 2 above)
Comment (jailton) [reply content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html]: Done!

File: content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html (hunk 1 above)
Comment (jcjesus) [reply content/pt/docs/tutorials/kubernetes-basics/explore/explore-interactive.html:32]: mmmm, ok! Thanks!

**Review by jcjesus — APPROVED** [review]

File: content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                <ul>
+                    <li>Armazenamento compartilhado, como Volumes</li>
+                    <li>Rede, como um endereço IP único no cluster</li>
+                    <li>Informações sobre como executar cada contêiner, como a versão da imagem do contêiner ou portas específicas a serem usadas</li>
+                </ul>
+                <p>Um Pod define um "host lógico" específico para o aplicativo e pode conter diferentes contêineres que, nas maiorias dos casos, são fortemente acoplados. Por exemplo, um Pod pode incluir o contêiner com seu aplicativo Node.js, bem como um outro contêiner que alimenta os dados a serem publicados pelo servidor web Node.js. Os contêineres de um Pod compartilham um endereço IP e intervalo de portas; são sempre localizados, programados e executam em um contexto compartilhado no mesmo Nó.</p>
```
Comment (rikatz) [inline content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html:38]: na maioria dos casos (está nas)

File: content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_03_pods.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2>Nós (Nodes)</h2>
+                <p>Um Pod sempre será executando em um <b>Nó</b>. Um Nó é uma máquina operária em um cluster Kubernetes e pode ser uma máquina física ou virtual. Cada Nó é gerenciado pelo Control Plane. Um Nó pode possuir múltiplos Pods e o Control Plane do Kubernetes gerencia automaticamente o agendamento dos Pods nos nós do cluster. Para o agendamento automático dos Pods, o Control Plane leva em consideração os recursos disponíveis em cada Nó.</p>
```
Comment (rikatz) [inline content/pt/docs/tutorials/kubernetes-basics/explore/explore-intro.html:77]: ao invés de máquina operária, eu sugiriria 'uma máquina de processamento'.

Até porque 'operária' vai de encontro com a tradução em inglês de "worker" que acho que o wg-naming quer remover :)

**Comment by rikatz** [comment]
/lgtm

Thanks / Obrigado!

**Pull request merged**